keps/sig-scheduling/20200612-deducted-topology-manager.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Deducted version of TopologyManager in kube-scheduler
```

Comment:
Is "Tailored" more proper? (Not an English expert though..)

Reply:
node-topology aware scheduling

Reply:
The meaning here was that we implement reduced/simplified version of topology manager.
Yes maybe better Node-topology aware scheduling - it's both a specific and an abstract.